scripts/gen_mmu.py

```diff
@@ -33,36 +40,941 @@
 FourMB = (1024*4096) #In Bytes
 #############
 
-#return the page directory number for the give address
-def get_pde_number(value):
-    return( (value >> 22 ) & 0x3FF)
+#*****************************************************************************#
+#class for 4Kb Mode
+class PageMode_4kb:
+    total_pages = 1023
+    write_page_entry_bin = "I"
+    size_addressed_per_pde = (1024 * 4096) #4MB In Bytes
+
+    #return the page directory number for the give address
+    def get_pde_number(self,value):
+        return( (value >> 22 ) & 0x3FF)
```

Comment:
I'd appreciate if [PEP-8](https://www.python.org/dev/peps/pep-0008/) were followed in this script, so we're consistent across the Zephyr code base.  There are some issues in this file regarding spacing, some stray semicolons, etc.

Reply:
yes agreed, let's get this consistent with PEP-8

Reply:
I was not aware of this standard. Will update accordingly.